README.md

```diff
@@ -54,61 +79,104 @@ CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s
+show how to retrieve this secret using the Conjur Java API once a connection is established. If you have a different use case, you can use the below policy as the basis for 
+another desired outcome. If a policy already exists for your host, you can skip the next set of steps.
+
+1. Make sure Conjur is running in the background and open the Conjur CLI.
+    
+    _NOTE:_ If using Docker: `docker-compose exec <NAME_OF_CLI_CONTAINER> bash`
 
-In addition, to run the integration tests you will need to load a Conjur policy. Save this file as `root.yml`:
+2. Copy the following policy, substitute the name of the policy, hostname, and permissions accordingly, and save it as a `.yml` file. 
 
-```yaml
+```xml
 - !policy
-  id: test
+  id: <POLICY_ID>
   body:
+    - !host
+      id: <NAME_OF_HOST>
+
     - !variable
-      id: testVariable
+      id: <NAME_OF_VARIABLE>
+
+    - !permit
+      role: !host <NAME_OF_HOST>
+      privileges: [read, execute]
+      resource: !variable <NAME_OF_VARIABLE>
 ```
 
-To load the policy, use the CLI command `conjur policy load root root.yml`
+3. Load the policy in the Conjur CLI like so: `conjur policy load <NAME_OF_POLICY>.yml`
 
-### SSL Certificates
+4. Once the policy is loaded (for the first time), an ID and API key will be returned for the host. It should look something like:
+```bash
+Loaded policy 'root'
+{
+  "created_roles": {
+    "cucumber:host:<POLICY_ID>/<NAME_OF_HOST>": {
+      "id": "cucumber:host:<POLICY_ID>/<NAME_OF_HOST>",
+      "api_key": "3scdjaz3aq90vn26avdv11s2mf2h2wnwfzw3cbgbja1jspzyc1rdzwqx"
+    }
+  },
+  "version": 1
+}
+```
+
+5. Set the ID and API key as the environment variables CONJUR_AUTHN_LOGIN and CONJUR_AUTHN_API_KEY respectively. 
+    
+    _NOTE:_ In Conjur, it is the standard that hosts are referred to in terms of their full hostname. Therefore, when setting value for the host's environment 
+    variable, the host ID should be prefixed with `host/`. For example, the `host/<NAME_OF_HOST>`, not just `<NAME_OF_HOST>`.
+
+6. Give the variable defined in the previous policy a value like so: `conjur variable values add <POLICY_ID>/<NAME_OF_VARIABLE>`
 
-By default, the Conjur appliance generates and uses self-signed SSL certificates. You'll need to configure
-Java to trust them. You can accomplish this by loading the Conjur certificate into the Java keystore.
+Now that we have identified our host in Conjur, we now need the Java app to trust Conjur. To do this, the Conjur self-signed SSL certificate needs to be added 
+to the app's keystore.  
+
+## Setup Trust Between App and Conjur
+By default, the Conjur appliance generates and uses self-signed SSL certificates (Java-specific certificates known as cacerts). 
+You'll need to configure your Java app to trust them. You can accomplish this by loading the Conjur certificate into the app's keystore.
```

Comment:
`Java's CA keystore that holds the list of all the allowed certificates for https connections.`